.github/repo-metadata.yml

```diff
@@ -0,0 +1,5 @@
+# manages metadata about the repository within Git
+
+milestones:
+  # assign new PRs to this milestone
+  next-milestone: 4.16.0
```

Comment:
Does it require manual update?

Reply:
That's right. Would be great to have this automated, but I don't see an obviously simple way

Reply:
I can write brute force python script for that :)

Reply:
The question is around the workflow... when to do this and such. We can just create a github workflow to update this file whenever a new milestone is created - but that wouldn't necessarily be right 🤔 do you have an idea about automating it in that regard?

Reply:
Hmm, I'll try to think how we can achieve that. But I guess this one can be merged anyways.

Reply:
Just gonna wait on our infrastructure folk @demonnic @keneanung to have a look first

Reply:
I think this will work for now, but we should include bumping the next milestone version as part of the release automation. Or as part of feature freeze automation, perhaps. And then we can adjust this to programatically determine the 'latest' milestone to assign PRs to. But no reason to wait on that to get this improvement in place.